src/Main.java

```diff
@@ -3,7 +3,96 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задача 1");
 
+        int savings = 15000;
+        int total = 0;
+        int month = 0;
+        while (total <= 2_459_000) {
+            month++;
+            total = total + total / 100;
+            total = total + savings;
 
+
+            System.out.println("Месяц " + month + " сумма накоплений равна " + total + " рублей");
+        }
+
+
+        System.out.println("Задача 2");
+        int j = 0;
+        while (j < 10) {
+            j++;
+            System.out.print(" " + j);
+        }
+        System.out.println(" ");
+        for (int i = 10; i >= 1; i = i - 1) {
+            System.out.print(" " + i);
+        }
+        System.out.println();
+
+        System.out.println("Задача 3");
+        int peopleWereBorn = 17;//Рождаемость
+        int peoleDied = 8; // Смертность
+        int population = 12_000_000; //Население
+        int populationYear; //Население за год
+        int result = 0;
+        for (int year = 2024; year <= 2034; year++) {
+            peopleWereBorn = population / 1000 * 17;
+            peoleDied = population / 1000 * 8;
```

Comment:
не используются переменные peopleWereBorn и peoleDied, вместо них числа

Reply:
также здесь надо было сразу записать:
```suggestion
            population = population + population * peopleWereBorn / 1000 - population * peoleDied / 1000;
```